Components/Navigation/styledNavigation.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+export const ButtonContainer = styled.div`
+  display: flex;
+  gap: 1rem;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 0.5rem;
+  margin-top: 1rem;
+`;
+
+export const StepperContainer = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: 0.3rem;
+`;
+
+export const SingleStepContainer = styled.section`
+  display: flex;
+  flex-direction: row;
```

Comment:
```suggestion
```

Reply:
Since `row` is the default value, it doesn't need to be defined.

Reply:
Done!